Source/WebKit/Shared/WebWheelEvent.cpp

```diff
@@ -32,7 +32,7 @@ namespace WebKit {
 
 using namespace WebCore;
 
-WebWheelEvent::WebWheelEvent(Type type, const IntPoint& position, const IntPoint& globalPosition, const FloatSize& delta, const FloatSize& wheelTicks, Granularity granularity, OptionSet<Modifier> modifiers, WallTime timestamp)
+WebWheelEvent::WebWheelEvent(WebEvent::Type type, const IntPoint& position, const IntPoint& globalPosition, const FloatSize& delta, const FloatSize& wheelTicks, Granularity granularity, OptionSet<Modifier> modifiers, WallTime timestamp)
```

Comment:
I'm curious why the argument type had to be more specific here (`WebEvent::Type` vs. `Type`) when `WebWheelEvent::isWheelEventType()` below didn't need to change.

Also, `WebMouseEvent::isMouseEventType()` and `WebKeyboardEvent::isKeyboardEventType()` didn't need to change in their respective source files.

To be clear, this isn't technically wrong, but I'm curious why it's not consistent with other similar methods.

Reply:
I think (but am not sure) that `WebWheelEvent` and the others inherit the changes from `WebEvent`? Not sure, but it's never asked me to change those across those files other than when it is using `WebEvent` directly in those files